index.yaml

```diff
@@ -108,7 +130,7 @@ entries:
     appVersion: "1.0"
     created: "2021-11-16T19:49:36.102667-08:00"
     description: A Helm chart for Lacework Proxy Scanner
-    digest: 652a7a24a1b93da1afe9ebcff66b08c79580b672e9a17a739c50ffa330a78f6d
+    digest: adc5ba1623b878b70329e67cf97263f7ff353befdd2d47efe8f2e75808d6ad76
```

Comment:
Digest changed because proxy scanner was repackaged in previous commit but index wasn't updated.

Reply:
Thanks for the note!